l10n_nl_xaf_auditfile_export/README.rst

```diff
@@ -0,0 +1,61 @@
+XAF auditfile export
+====================
+
+This module allows you to export XAF audit files for the Dutch tax authorities (Belastingdienst).
+
+The currently exported version is 3.2
+
+Usage
+=====
+
+To use this module, you need to:
+
+* go to `Invoicing`/`Reporting`/`Legal Reports`/`Auditfile export`
+* create a new record, adjust values if the defaults are not appropriate
+* click `Generate auditfile`
+* click `Download` on the field `Auditfile`
+
+For further information, please visit:
+
+* https://www.odoo.com/forum/help-1
+
+Configuration
+=============
+
+This module works on huge amount of data, so there is a possibility to encounter out of memory exceptions. In this case. set the config parameter `l10n_nl_xaf_auditfile_export.max_records` to a value much lower than 10000.
+
+Known issues / Roadmap
+======================
+
+* encrypted and compressed files would be nice
```

Comment:
I did nothing here because there's 7.0 in between. And depending on which implementation we choose there (either backport this which would have my preference or port nbs_xml_auditfile_financieel) we need a different migration